Makefile

```diff
@@ -42,11 +50,14 @@ clean:
 	for d in $(SUBDIRS); do $(MAKE) -C $$d clean; done
 	- rm -rf $(BINPATH)/
 	$(MAKE) -C $(RUNC_DIR) clean
+	$(MAKE) firecracker-clean
 	rm -f *stamp
 	$(MAKE) -C tools/image-builder clean-in-docker
 
 distclean: clean
-	docker rmi localhost/runc-builder:latest
+	docker rmi localhost/$(RUNC_BUILDER_NAME):$(DOCKER_IMAGE_TAG)
+	docker rmi localhost/$(FIRECRACKER_BUILDER_NAME):$(DOCKER_IMAGE_TAG)
+	docker volume rm -f $(CARGO_CACHE_VOLUME_NAME)
```

Comment:
these docker commands should be prefixed with '-' so they don't count as a failure if a given image doesn't exist.

Raise the bar by providing a make function that checks to see if an image exists, exits true if not, tries to delete it if so (at that point you can fail meaningfully if it doesn't exist)

Reply:
I learned some more Make and added a macro for conditionally deleting the images. For the docker volume, the "-f" gets us that effect already (there is also a -f flag for rmi but it does much more that we don't want).